guillotina/tests/memcached/test_memcached_driver.py

```diff
@@ -189,22 +197,20 @@ async def metrics(self):
         metrics.create_connection_upper = 100.0
         return metrics
 
-    async def test_updated_connection_pool_metrics_create_connection_latencies(
-        self, avg, p50, p99, upper, metrics
-    ):
+    def test_updated_connection_pool_metrics_create_connection_latencies(self, avg, p50, p99, upper, metrics):
```

Comment:
I think when the pytestmark `pytest.mark.asyncio` is set at module level, all tests must be async.

Reply:
yes